landing page blog (#71)
Contenda-Team/Contenda-landing-page: src/posts/landingpage.md

---
layout: "../layouts/BlogPost.astro"
title: "How Astro and Rive Supercharged Our Landing Page"
slug: landingpage
description: "for anyone interested in efficient design-to-development workflows"
author: "Sally Chen"
added: "June 13 2023"

> tags: ["user experience, frontend"]

Reviewer: ```suggestion
tags: ["user experience", "frontend", "design"]
```

---

> 

Reviewer: I think you should introduce yourself and what your purpose is

You may have noticed that Contenda’s website has undergone quite the transformation recently. This post will give you a sneak peek into the behind-the-scenes work that went into this redesign. Discover how I used Astro and Rive to conceptualize, enhance, and even implement my designs!

> ## Astro & Rive: The Dynamic Duo

Reviewer: Given that the example is just about Figma/Rive, I think you should remove all the Astro references? It doesn't seem relevant to anything else in this blog.

[Astro](https://astro.build/), a super versatile JavaScript framework, provides a blend of flexibility and performance while catering to those of us who appreciate working with the familiar realms of HTML and CSS.

> This meant that even with little familiarity regarding advanced frontend technologies, I could still code out my designs and immediately see them come to life.

Reviewer 2: small thoughts. 
"advanced" makes it sound like astro isn't so maybe change to "mainstream"
adding that you specifically used astro components for the plain html/css components and how that works with our react components just for more tech info bits.
```suggestion
This meant that even with little familiarity regarding mainstream frontend technologies, I could still code out my designs and immediately see them come to life through Astro components. With my Astro components living seamlessly along side our existing React components, Astro allowed the flexibility to choose how to build in a way that was most comfortable for me. 
```

[Rive](https://rive.app/) is a game-changer for creating interactive animations that are platform-agnostic – meaning they can run on any device, on any platform. It offers an intuitive interface, similar to design platforms like Figma, which makes the animation process more streamlined. Plus, its embeddable iframe export options merge seamlessly into our Astro codebase. 

> ## Astro and Rive in Action: Bringing the Contenda Kitchen to Life

Reviewer 3: i think calling it the newsletter is better here and also i really did not like the spacing of it LOL on my screen it was just `to Life` on the second line. not sure if this fixes it but 

```suggestion
## Astro and Rive in Action: Bringing the Contenda Newsletter to Life
```

To create the fortune cookie graphic, I turned to Figma and used vector graphics. The next step was to breathe life into this static image. I exported the SVG from Figma and imported it into Rive. The striking similarity between Rive's interface and Figma's, right down to the preservation of Figma groupings, made the animation process a breeze. With a few tweaks and movements on the Rive canvas, I had an animated graphic ready to go. Exporting the animation as a one-line iframe embed code was the cherry on top!

The last step was integrating this animation into the landing page. With Astro, all it took was some rearrangement of divs in the codebase and pasting a single iframe embed line. Astro's compatibility with HTML and CSS made this process straightforward and intuitive.

![Adding a Rive animation was that easy!](/blogimages/Rive-Embed.png)

## So, there you have it -

> A peek behind Contenda’s new landing page! Thanks to the power and flexibility of Astro and Rive, my journey from design to implementation was not only efficient, but also enjoyable. Stay tuned for more update!

Reviewer: I think you should tell your narrative more at the beginning so that this ending is satisfying
